vtex/toolbelt
# Create 'vtex test e2e' command

#### What is the purpose of this pull request?
Creating MVP for allowing developers to run e2e test in the Vtex IO ecosystem using cypress with a simple toolbelt command.

#### What problem is this solving?
A unified way to run e2e tests in the Vtex IO ecosystem using cypress.

#### How should this be manually tested?
The toolbelt `vtex e2e` command can be used to run a full e2e test for an app with cypress tests

the toolbelt must be using this branch
Also builder-hub, tester-hub and tester-e2e-cypress must be linked in the respective branches:
https://github.com/vtex/builder-hub/tree/feature/e2e_builder_for_cypress
https://github.com/vtex/tester-hub/tree/test/meta
https://github.com/vtex/tester-e2e-cypress/tree/feature/cypress

first link the builder-hub, then link the tester-hub and the tester-e2e-cypress.
Use the command yarn watch in the toolbelt
And then you can use vtex-test e2e in another instance of the tester-hub (or any other app with cypress test if you link it first)

the results should be the same as in running the cypress test with `yarn run cypress open`
#### Screenshots or example usage

#### Types of changes

#### Chores checklist
